debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/MongoDbReplicaSet.java

```diff
@@ -45,7 +44,9 @@
 public class MongoDbReplicaSet implements MongoDbDeployment {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MongoDbReplicaSet.class);
-
+    private static final String STARTUP_TIMEOUT = System.getProperty("startup.timeout.seconds");
```

Comment:
I wonder if it would be best to prefix this with `debezium.test.` like we do for other wait times just so that we keep some consistency with how these are named.  wdyt @ani-sha ?

Reply:
@Naros I had initially added the prefix, but since it was not directly used in the test classes, I removed it. I agree, it's good to have the naming consistency throughout the code. Do you have any other prefix in mind or we should go with `debezium.test.`?

Reply:
I would add some reference to better identify `startup of what`? Something like ` debezium.test.mongo.replica.primary.startup.timeout.seconds` WDYT?

Reply:
I agree @mfvitale, we should try and aim at being descriptive :100: